sdk/src/main/java/com/mongodb/stitch/android/auth/AuthInfo.java

```diff
@@ -5,18 +5,24 @@
 import com.fasterxml.jackson.annotation.JsonProperty;
 
 /**
- * Auth represents a session on Stitch for a specific user.
+ * AuthInfo represents a session on Stitch for a specific user.
```

Comment:
Reminder that this will cause a major SEMVER bump

Reply:
Yep.